observation_portal/accounts/forms.py

```diff
@@ -27,7 +27,7 @@ class Meta:
         }
 
     def save(self, commit=True):
-        new_user_instance = super().save(commit)
+        new_user_instance = super().save(commit=True)
```

Comment:
This change is from this commit by Jon: https://github.com/observatorycontrolsystem/observation-portal/commit/d38870f9dfac56416ea4dfe26617549338ceee2c

Reply:
@jnation3406 I assume this is cause they changed it to be a kwarg. Should we instead do `new_user_instance = super().save(commit=commit)` to use the value of `commit` that is passed in? Or should it always just be `True`?

Reply:
I believe it was causing issues with commit not being True. There was some issue where commit was not true, so the new user wasn't actually created so the next line of adding it into the profile wasn't working properly. I forget what the exact error/issue was at this point though...